src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int RANDOM_FIGURES_ARRAY_SIZE = 3;
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] randomFiguresArray = new Figure[RANDOM_FIGURES_ARRAY_SIZE];
+        randomFiguresArray[RANDOM_FIGURES_ARRAY_SIZE - 1] = figureSupplier.getDefaultFigure();
+        for (int i = 0; i < (randomFiguresArray.length - 1); i++) {
+            randomFiguresArray[i] = figureSupplier.getRandomFigure();
+        }
+        for (int i = 0; i < randomFiguresArray.length; i++) {
+            randomFiguresArray[i].draw();
+        }
+    }
```

Comment:
It's better to use 1 cycle for all logic and also make code dependant on single constant value in order to change only 1 number in code to make it work for 8, 10 or 999 balls properly
```suggestion
    public static void main(String[] args) {
        Figure[] randomFiguresArray = new Figure[RANDOM_FIGURES_ARRAY_SIZE];
        for (int i = 0; i < (randomFiguresArray.length); i++) {
            if (i < randomFiguresArray.length / 2) {
                randomFiguresArray[i] = figureSupplier.getRandomFigure();
            } else {
                randomFiguresArray[i] = figureSupplier.getDefaultFigure();
            }
            randomFiguresArray[i].draw();
        }
    }
```